Appraisals

```diff
@@ -2,12 +2,15 @@
 
 appraise 'rails-4' do
   gem 'activerecord', '~> 4.2'
+  gem 'sqlite3',      '~> 1.3.6'
```

Comment:
Given the above comment about the use of `declare_schema` I thought it helpful to synchronize the appropriate sqlite/ activerecord versions to ensure parity. [4.2](https://github.com/rails/rails/blob/4-2-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L5) [5.2](https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L12) and [6.0](https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L13)

Reply:
Ah, adding this isn’t necessary, because activesupport itself adds this dependency and locks the version implicitly